mkdocs/topics/deployments.md

```diff
@@ -34,9 +31,97 @@ It's highly recommended that you create a staging version of your project anytim
 
 </div>
 
+### Mint
+Mint is a great way to deploy Visivo. It's caching functionality, concurrency and debugging can make development of Mint workflows much easier than Git Actions. The workflows below will deploy a stage when the pull request is updated and archive it when the pull request is closed. 
+
+#### Configuration 
+
+1. Add your deployment token as `VISIVO_TOKEN` to your [Mint vault](https://www.rwx.com/docs/mint/vaults#vaults).
+2. Add a workflow similar to the following yml. 
+
+!!! note 
+
+    The example below is for a project that runs Visivo against google cloud SQL. You will likely need to pass additional secrets and alter authentication steps to connect to your targets.
+
+{% raw %}
+``` yaml title=".mint/manage_ci_gcp_cloud_sql_target.yml"
+on:
+  github:
+    pull_request:
+      actions: [opened, reopened, synchronize, closed]
+      init:
+        commit-sha: ${{ event.github.pull_request.pull_request.head.sha }}
+        head-ref: ${{ event.github.pull_request.pull_request.head.ref	}}
+        deploy: ${{ event.github.pull_request.pull_request.merged == false && event.github.pull_request.pull_request.state == 'open' }}
+        archive: ${{ event.github.pull_request.pull_request.merged == true && event.github.pull_request.pull_request.state == 'closed' }}
+
+tasks:
+  - key: code
+    call: mint/git-clone 1.1.6
+    with:
+      repository: https://github.com/visivo-io/analytics.git
+      ref: ${{ init.commit-sha }}
+      github-access-token: ${{ secrets.VISIVO_IO_CLONE_TOKEN }} #(3)!
+  
+  - key: python
+    call: mint/install-python 1.0.2
+    with:
+      python-version: 3.10.0
+  
+  - key: install-visivo
+    use: [python]
+    run: pip install git+https://github.com/visivo-io/visivo.git@latest #(4)!
+  
+  - key: gcloud-cli # (1)!
+    call: google-cloud/install-cli 1.0.1
+  
+  - key: gcloud-login # (2)!
```

Comment:
I think I would break out the cloud SQL into a different item because I think most people won't need this.

Reply:
Yeah that's a good point. I'll tab them out with the basic script showing by default.

Reply:
Done 👍